### ISSUE TITLE
Apply accessibility improvements that were made to design system

### ISSUE THREAD

**Issue by kkoryaka** [cagov/cannabis.ca.gov#467]
Update packages:
ds-button-grid to version 2.0.2
ds-content-navigation to version 2.0.0
ds-dropdown-menu to version 2.0.1
ds-page-alert to version 1.0.2
ds-statewide-footer to version 1.0.7
ds-step-list to version 2.0.1

Add aria-hidden attribute to statewide header and statewide ca.gov icon
Add aria-controls to mobile search button (and add id reference to mobile search container) 
make sure that site is using npm component packages instead of local

**Comment by kkoryaka** [comment]
@yesenia-gtz i think this issue should be closed. This one was related to the cannabis project
